src/join.t.sol

```diff
@@ -701,6 +702,60 @@ contract DssDeployTest is DssDeployTestBase {
         gusdJoin.exit(address(this), 10 * 10 ** 2);
     }
 
+    function testFailManagedGemJoinJoinWad() public {
+        deployKeepAuth();
+        DSValue pip = new DSValue();
+        WBTC wbtc = new WBTC(100 * 10 ** 2);
```

Comment:
Kind of an odd way to express this--why not `10_000` or `10 * 10 ** 3`? `WBTC` (in this repo at least) has `decimals = 8` so it's not like `10 ** 2` is a natural unit for the total supply.

Reply:
Right, changed to 10 * 8.